frontend/vue.config.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = {
+  devServer: {
+    proxy: {
+      '^/api': {
+        target: 'http://localhost:8081',
```

Comment:
Hier sollte vor einem PR der Bezug auf den Localhost entfernt werden. Dein Backend sollte auf dem Server unter http://server.it-humke.de:9002 erreichbar sein